src/components/Converter.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import '../styles/styles.scss';
+
+class Converter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            x: this.props.x ? this.props.x : '',
+            y: this.props.y ? this.props.y : '',
+            rate: this.props.rate ? this.props.rate : 1,
+        };
+    }
+
+    roundIt(num) {
+        return Math.round(num * 1000) / 1000;
+    }
+
+    handleChange(e) {
+        if (!e.target.value) {
+            this.setState({ x: '', y: '' });
+            return;
+        }
+
+        let value = parseFloat(e.target.value);
+        if (e.target.id === 'x')
+            this.setState({
+                x: value,
+                y: this.roundIt(value * this.state.rate),
+            });
+        else if (e.target.id === 'y')
+            this.setState({
+                x: this.roundIt(value / this.state.rate),
+                y: value,
+            });
+    }
+
+    render() {
+        let xLabel = this.props.xLabel ? this.props.xLabel : '1';
+        let yLabel = this.props.yLabel || '1';
+        return (
+            <div className='converter'>
+                <h4 className='converter-heading'>
```

Comment:
in your styles.scss you are nesting just `.heading` and not `converter-heading` for the className inside the `converter` class.